todo/src/components/todoForm.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux'
+import { addTask, toggleTask } from '../actions'
+
+
+class TodoForm extends Component {
+  constructor(props){
+    super(props);
+    this.state={
+      todoInput: '',
+    }
+  }
+  
+  toggleTask = id => {
+    this.props.toggleTask(id)
+  }
+  addTask = (e) => {
+    e.preventDefault();
+    this.props.addTask(this.state.todoInput);
+    this.setState({
+      todoInput: ''
+    })
+  }
+  handleInputChange = e => {
+    console.log('Input value',e.target.value)
+    this.setState({
+      [e.target.name]: e.target.value
+    })
+  }
+  render() {
+    const completedStyle = {
+      fontStyle: 'italic',
+      color: "#cdcdcd",
+      textDecoration: "line-through"
+  }
+    return (
+      <div className='container'>
+        <h3 className='header'>Path to World Domination</h3>
+        <div className='todoList'>
+        {this.props.todoStateFromProps.map(todo => <p style={todo.completed ? completedStyle : null} onClick={() => this.toggleTask(todo.id) }>{todo.value}</p>)}
```

Comment:
Try avoiding using `style`, try to do a className instead, and manage all the styling in CSS. You can do a ternary operator on classes too